csrc/scheduler/pointwise.cpp

```diff
@@ -350,9 +349,10 @@ std::unique_ptr<PointwiseParams> getPointwiseHeuristics(
 
   auto broadcast_info = HeuristicDataCacheEntry<
       HeuristicCompileTime::BroadcastMultiples>(
-      data_cache, [&largest_out, &index_type]() {
+      data_cache, [&largest_out, &index_type, &reorder_map]() {
```

Comment:
@jjsjann123 looked like we should have passed the reorder map to this call as we do to `getVectorizationFactor` and also reorder the loop. I am understanding `getBroadcastMultiples` to come up with a test case which will break without it.
Correct me if I am wrong though, and it was intentional.